### PR TITLE
implement i18n

### PR DESCRIPTION
**Pull request by MuhammadTamzid**

### CODE REVIEW

**Comment by shakil-fg** [comment]
ok

**Comment by mdmahmudurrahman** [comment]
Ok

**Comment by sujoydatta** [comment]
ok

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,5 +1,12 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
+  before_action :set_locale
   protect_from_forgery with: :exception
+
+  def set_locale
```
Comment (dieunb) [inline app/controllers/application_controller.rb]: private function

**Pull request merged**